docs/current_docs/manuals/developer/services.mdx

```diff
@@ -203,6 +330,9 @@ Here's what happens on the last line:
 1. Dagger waits for health checks to pass against `redisSrv`.
 1. Dagger runs the `ping` container with the `redis-srv` alias magically added to `/etc/hosts`.
 
+</TabItem>
+</Tabs>
+
 :::note
 Dagger cancels each service run after a 10 second grace period to avoid frequent restarts, unless the explicit `Start` and `Stop' APIs are used.
```

Comment:
```suggestion
Dagger cancels each service run after a 10 second grace period to avoid frequent restarts, unless the explicit `Start` and `Stop` APIs are used.
```

Reply:
Please feel free to merge after the above fix